WordPress/gutenberg
# Package `@ariakit/test` should be a dev dependency

## What?

Remove an unnecessary dependency of the published `@wordpress/components` package on `@ariakit/test`.

## Why?

As it is, everyone using `@wordpress/components` winds up having to install `@ariakit/test` and its dependencies unnecessarily.

## How?

Move the dependency to `devDependencies`. And, per your linting, apparently devDependencies has to go in the root package.json.

## Testing Instructions
With `pnpm`, you can do `pnpm add file:///path/to/gutenberg/packages/components` in a fresh directory and then check that `@ariakit/test` is not installed as a dependency.

Looks like with npm you'll have to `npm pack` to create a `wordpress-components-25.11.0.tgz` file then in the fresh directory `npm install /path/to/wordpress-components-25.11.0.tgz`. Then, again, check that `@ariakit/test` is not installed as a dependency.
